configuration/scripts/machines/Macros.conda_macos

```diff
@@ -45,6 +45,7 @@ ifeq ($(strip $(SDKPATH)),)
   CFLAGS_HOST := 
 else
   CFLAGS_HOST = -isysroot $(SDKPATH)
+  LD += -L$(SDKPATH)/usr/lib
```

Comment:
I'm a little curious about why this is suddenly needed, but I guess something changed in the conda-forge toolchain...

Reply:
I don't know.  This is what I had to do on my new mac running 11.2.1.  I hadn't been able to run the conda macos before on this machine (although I had quickly tried before).  I also had to make sure I updated my conda environment because I had an older install that had the omp bug.  The extra LD could be a bug in the underlying conda setup, but I personally don't really know enough to be sure.